bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Gateway.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Gateway
+{
+    /// <summary>
+    ///  Wrapper for Azure IoT Gateway SDK. 
+    /// </summary>
+    public class Gateway
+    {
+        [DllImport(
+            "gateway",
+            CharSet = CharSet.Ansi,
+            EntryPoint = "Gateway_CreateFromJson",
+            CallingConvention = CallingConvention.Cdecl
+        )]
+        private static extern IntPtr CreateFromJsonInternal(string file_path);
+
+
+        /// <summary>
```

Comment:
I think on this case we should duplicate...... I don't knwo how I feel about adding links on public method headers (If they are in Doxygen it's ok, but if they are reading on the cs itself it won't look good. 
So, I think it's useful the documentation here. 

---
In reply to: [104810625](https://github.com/Azure/azure-iot-gateway-sdk/pull/151#discussion_r104810625) [](ancestors = 104810625)